docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -0,0 +1,342 @@
+
+#### `Notify=` (defaults to `no`)
+
+By default Podman is run in such a way that systemd startup notify is handled by the container
+runtime. In other words, the service is deemed started when the container runtime starts the
+child in the container. However, if the container application supports
+[sd_notify](https://www.freedesktop.org/software/systemd/man/sd_notify.html) then setting
+`Notify`to true will pass the notification details to the container allowing it to notify
+of startup on its own.
+
+#### `Timezone=` (if unset uses system-configured default)
+
+The timezone to run the container in.
+
+#### `RunInit=` (default to `yes`)
+
+If enabled (and it is by default), the container will have a minimal init process inside the
+container that forwards signals and reaps processes.
+
+#### `VolatileTmp=` (default to `yes`)
+
+If enabled (and it is by default), the container will have a fresh tmpfs mounted on `/tmp`.
+
+**NOTE:** Podman will automatically copy any content from the image onto the tmpfs
+
+#### `Volume=`
+
+Mount a volume in the container. This is equivalent to the Podman `--volume` option, and
+generally has the form `[[SOURCE-VOLUME|HOST-DIR:]CONTAINER-DIR[:OPTIONS]]`.
+
+As a special case, if `SOURCE-VOLUME` ends with `.volume`, a Podman named volume called
+`systemd-$name` will be used as the source, and the generated systemd service will contain
+a dependency on the `$name-volume.service`. Such a volume can be automatically be lazily
+created by using a `$name.volume` quadlet file.
+
+This key can be listed multiple  times.
+
+#### `ExposeHostPort=`
+
+Exposes a port, or a range of ports (e.g. `50-59`) from the host to the container. Equivalent
+to the Podman `--expose` option.
+
+This key can be listed multiple  times.
+
+#### `PublishPort=`
+
+Exposes a port, or a range of ports (e.g. `50-59`) from the container to the host. Equivalent
+to the Podman `--publish` option. The format is similar to the Podman options, which is of
+the form   `ip:hostPort:containerPort`, `ip::containerPort`, `hostPort:containerPort` or
+`containerPort`, where the number of host and container ports must be the same (in case
+of a range).
+
+If the IP is set to 0.0.0.0 or not set at all, the port will be bound on all IPv4 addresses on
+the host, use [::] for IPv6.
+
+Note that not listing a host port means that Podman will automatically select one, and it
+may be different for each invocation of service. This makes that a less useful option.  The
+allocated port can be found with the `podman port` command.
+
+This key can be listed multiple  times.
+
+#### `PodmanArgs=`
+
+This key contains a list of arguments passed directly to the end of the `podman run` command
+in the generated file (right before the image name in the command line). It can be used to
+access Podman features otherwise unsupported by the generator. Since the generator is unaware
+of what these options to it can cause unexpected interactions, so it is not recommended to use
```

Comment:
```suggestion
Since the generator is unaware of what unexpected interactions can be caused by these arguments, it is not recommended to use this option.
```